CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
-
+- Added in WindowAttributes `center_window: bool` `start_monitor: i16`, Window centering is by default enabled on windows.
```

Comment:
"Added to \`WindowAttributes\` the attributes  \`center_window: bool\` and \`start_monitor: i16`. Window centering is by default enabled on windows."